controllers/grid/files/galley/GalleyFilesGridHandler.inc.php

```diff
@@ -107,42 +93,12 @@ function initialize($request) {
 		);
 
 		// Basic grid configuration
-		$this->setId('articleGalleyFiles-' . $this->getAssocId());
+		$representation = $this->getAuthorizedContextObject(ASSOC_TYPE_REPRESENTATION);
+		$this->setId('articleGalleyFiles-' . $representation->getId());
 		$this->setTitle('submission.galleyFiles');
 		$this->setInstructions('submission.proofReadingDescription');
 	}
 
-	/**
-	 * @copydoc SignoffFilesGridHandler::getRowInstance()
-	 */
-	protected function getRowInstance() {
-		$row = parent::getRowInstance();
-		$row->setRequestArgs($this->getRequestArgs());
-		return $row;
-	}
-
-	/**
-	 * @copydoc GridHandler::getRequestArgs()
-	 */
-	function getRequestArgs() {
-		return array_merge(
-			parent::getRequestArgs(),
-			array('representationId' => $this->getAssocId())
-		);
-	}
```

Comment:
if we require a valid representation to valid the requests, why not publish the representation id via this interface? then all grid requests would have it already...